web-components/button/button.svelte

```diff
@@ -27,11 +61,12 @@
   .leoButton {
     // Gradients cannot have a transition, so we need to reset `transition`
     // to only apply to `box-shadow` and `border-color` in .isCTA
-    --default-transition: box-shadow .12s ease-in-out, color .12s ease-in-out, border-color .12s ease-in-out;
+    --default-transition: box-shadow 0.12s ease-in-out, color 0.12s ease-in-out,
+      border-color 0.12s ease-in-out;
```

Comment:
nit: don't love this formatting, but I'm assuming it's Prettier being opinionated?

Reply:
TBH, I don't even know if I'm using prettier. I'll revert this.